homeassistant/components/duotecno/entity.py

```diff
@@ -41,6 +41,11 @@ async def _on_update(self) -> None:
         """When a unit has an update."""
         self.async_write_ha_state()
 
+    @property
+    def available(self) -> bool:
+        """Available state for the unit."""
+        return self._unit.is_available()
```

Comment:
Assuming `_on_update` is being called when the device state changes, it would probably be better to assign `self._attr_available = self._unit.is_available()` in there?

Btw, it's a good practise to use properties instead of methods for information that do not cause I/O, this makes it easy to see if something is safe in contexts where no I/O is wanted. So you could do something like this in your library in the future, no need to change it for this though.
```
@property
def is_available(self):
    return self._is_available
```
```